api/v3/Entity.php

```diff
@@ -14,6 +14,8 @@ function _civicrm_api3_entity_deprecation($entities) {
   $deprecated = [];
   if (!empty($entities['values'])) {
     foreach ($entities['values'] as $entity) {
+      $apiFile = "api/v3/$entity.php";
+      @include_once $apiFile;
```

Comment:
The big performance gain from this PR will be *not* scanning for api files on every request, but only doing it in the one place where it's actually needed (which is a call that's basically only ever done by the api explorer).

But here's another little boost. People complain about the slow-ness of using the `@` prefix, but `isIncludable()` uses `@fopen()` (pointlessly because then it just `fclose`s and exits). So we might as well just do that in one-line here and skip that extra check.